Gemfile.lock

```diff
@@ -334,5 +334,8 @@ DEPENDENCIES
   webmock (~> 3.0.1)
   yard
 
+RUBY VERSION
+   ruby 2.5.3p105
```

Comment:
is there a reason why this been added?

Reply:
This was most likely added automatically when I `bundle`d. Can remove.